World.py

```diff
@@ -63,6 +64,19 @@ def __init__(self, world_id: int, settings: Settings, resolve_randomized_setting
         self.settings: Settings = settings.copy()
         self.distribution: WorldDistribution = settings.distribution.world_dists[world_id]
 
+        # language property...
+        self.language: Language = Language(settings.language_selection)
+        def update_dict(A: dict, B: dict) -> dict:
+            C = {}
+            for key, subdict in A.items():
+                merged = subdict.copy()
+                if key in B:
+                    merged.update(B[key])
+                C[key] = merged
+            return C
```

Comment:
The variable names should be in lowercase.
```suggestion
        def update_dict(a: dict, b: dict) -> dict:
            c = {}
            for key, subdict in a.items():
                merged = subdict.copy()
                if key in b:
                    merged.update(b[key])
                c[key] = merged
            return c
```